### ISSUE TITLE
use with sessionless JWT

### ISSUE THREAD

**Issue by alexandrepa** [drudge/passport-twitter-token#21]
I just want to know if it's possible to use this strategy without passport session enabled ?
Thanks

**Comment by jebarjonet** [comment]
@alexandrepa I don't think you can. This is exactly the same issue here https://github.com/jaredhanson/passport-twitter/issues/90 (and multiple other issues in this repo), OAuth 1 needs you to keep in memory a secret during the authentication. This repo and `passport-twitter` are both using [passport-oauth1](https://github.com/jaredhanson/passport-oauth1) which is based on sessions to store that secret... That's pretty unfortunate
